spec/unit/Onebip/StopWatch/MicrotimeClockStopWatchTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace Onebip\StopWatch;
+
+use Onebip\Clock\FixedMicrotimeClock;
+
+class MicrotimeClockStopWatchTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        $this->clock = new FixedMicrotimeClock(42.32);
+        $this->stopWatch = new MicrotimeClockStopWatch($this->clock);
+    }
+
+    public function testElapsedWithoutStarting()
+    {
+        $this->assertSame(0.0, $this->stopWatch->elapsedMicroseconds());
+        $this->assertSame(0.0, $this->stopWatch->elapsedMilliseconds());
+        $this->assertSame(0.0, $this->stopWatch->elapsedSeconds());
+    }
+
+    public function testElapsedWithoutStopping()
+    {
+        $this->stopWatch->start();
+
+        $this->assertSame(0.0, $this->stopWatch->elapsedMicroseconds());
```

Comment:
isn't safer to throw an exception here instead of returning a `0.0` which could be logged and used in other places before blowing up?